salvo/src/lib/benchmark/test_fully_dockerized_benchmark.py

```diff
@@ -2,25 +2,81 @@
 Test the fully dockerized benchmark class
 """
 
-import site
 import pytest
+from unittest import mock
 
 from api.control_pb2 import JobControl
 from src.lib.benchmark.fully_dockerized_benchmark import Benchmark
 
 
 def test_images_only_config():
-  """Test benchmark validation logic."""
-  pass
+  """Test benchmark validation logic.
 
+  Validate that we attempt to run the specified docker image
+  when all parameters are present.
+  """
+  # create a valid configuration defining images only for benchmark
+  job_control = JobControl()
+  job_control.remote = False
+  job_control.scavenging_benchmark = True
+
+  docker_images = job_control.images
+  docker_images.reuse_nh_images = True
+  docker_images.nighthawk_benchmark_image = \
+      "envoyproxy/nighthawk-benchmark-dev:test_images_only_config"
+  docker_images.nighthawk_binary_image = \
+      "envoyproxy/nighthawk-dev:test_images_only_config"
+  docker_images.envoy_image = \
+      "envoyproxy/envoy-dev:f61b096f6a2dd3a9c74b9a9369a6ea398dbe1f0f"
+
+  env = job_control.environment
+  env.variables["TMP_DIR"] = "/home/ubuntu/nighthawk_output"
```

Comment:
(optional) Are we planning to make the output directory value a configurable option?

Reply:
`TMP_DIR` is set to the output directory specified in the job control object.  It is configurable.